code/modules/mob/living/carbon/human/species_types/diona.dm

```diff
@@ -0,0 +1,48 @@
+/datum/species/dionae
+	name = "Dionae"
+	id = "diona"
+	blacklisted = FALSE
+	sexes = FALSE
+	say_mod = "rustles"
+	default_color = "228822"
+
+
+	inherent_biotypes = list(MOB_ORGANIC, MOB_HUMANOID)
+	mutant_bodyparts = list("diona_body_markings")
+	default_features = list("diona_body_markings" = "Default")
+
+	mutanttongue = /obj/item/organ/tongue/diona
+
+	meat = /obj/item/reagent_containers/food/snacks/meat/slab/human/mutant/lizard
+	skinned_type = /obj/item/stack/sheet/animalhide/lizard
+
+	liked_food = SUGAR | RAW | MEAT
+	disliked_food = TOXIC | JUNKFOOD
+
+	exotic_blood = "radium" //R A D I O A C T I V E T R E E. WHAT COULD POSSIBLY GO WRONG? MAYBE CHLORINE AND PHOSPHORUS.
+
+	no_equip = list(SLOT_GLOVES, SLOT_SHOES) //HOW THE FUCK ARE YOU SUPPOSED TO FIT ANYTHING OVER THOSE GIANT, ASS-GROPING TREE-TRUNK HANDS AND FEET?
+
+	//GIMMICK RACE PART 1: IT BEGINS.
+	speedmod = 8 //SUPER SLOW BECAUSE THEY'RE A TREE.
```

Comment:
A speed penalty like this will probably make the race too tedious to play as (for comparison, the plasmaman's penalty is 1) and unwanted by many.
Also, these all caps commentary is absolutely embarassing.